bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -27,7 +27,10 @@ import edu.gemini.spModel.core.{Angle, MagnitudeBand}
 import scalaz._
 import Scalaz._
 
-object GemsStrategy extends AgsStrategy {
+trait GemsStrategy extends AgsStrategy {
+  // By default use the remote backend but it can be overriden in tests
+  private [impl] implicit val backend:VoTableBackend = RemoteBackend
```

Comment:
@tpolecat I agree but it might be okay as a backdoor for testing that doesn't clutter "normal" usage?  The alternative is to pass in `RemoteBackend` explicitly everywhere `GemsVoTableCatalog.search` is called I suppose.